README.rst

```diff
@@ -1580,6 +1580,8 @@ when formatting is applied. The following options are available:
 +--------------------+----------+---------------+------------------------------+
 | ``json.sort_keys``            | ``true``      | ``--sorted``, ``--unsorted`` |
 +--------------------+----------+---------------+------------------------------+
+| ``response.content_type``     | ``automatic`` | ``--response-content-type``  |
++--------------------+----------+---------------+------------------------------+
```

Comment:
This feature will need an explanation in the docs. I’m also not sure about the naming — what this allows you to do is to "overwrite the response’s `Content-Type` for output formatting purposes". It’s a tricky one to capture under a short name, but let’s think about this a bit more.

Reply:
And it is already too long IMO. I am not sure adding more keywords would be better :thinking:

Reply:
I think "formatting purposes" and "overwrite" should not be part of such option."
Same as `--sorted` does not reflect that, else we would end up with `--force-sorting-response-and-headers-json-keys`. I made it to make my point, but indeed such specialized option could be renamed. Let's think about it.

Reply:
Yeah, I would definitely not make it longer.